src/components/data-exploration/differential-expression-tool/DiffExprResults.jsx

```diff
@@ -13,6 +13,7 @@ import _ from 'lodash';
 import GeneTable from '../generic-gene-table/GeneTable';
 import { geneTableUpdateReason } from '../../../utils/geneTable/geneTableUpdateReason';
 import loadDifferentialExpression from '../../../redux/actions/differentialExpression/loadDifferentialExpression';
+import { getCellSets } from '../../../redux/selectors';
```

Comment:
you can use the opportunity to get rid of those relative imports -- Martin fixed the linter last week, so it should work